collection-scripts/gather_namespaced_resources

```diff
@@ -91,6 +91,11 @@ oc_get_all+=("networkfence")
 oc_get_all+=("network-attachment-definitions")
 oc_get_all+=("encryptionkeyrotationjob")
 oc_get_all+=("encryptionkeyrotationcronjob")
+oc_get_all+=("operatorconfig")
+oc_get_all+=("driver")
+oc_get_all+=("cephconnection")
+oc_get_all+=("clientprofile")
+oc_get_all+=("clientprofilemapping")
```

Comment:
please use the full names as these are very generic names we might get into problem, if we have duplicate CRD with name short names